app/points/domain/tier-points-configuration.js

```diff
@@ -0,0 +1,26 @@
+const assertNumber = require('./validation/assert-number')
+
+class TierPointsConfiguration {
```

Comment:
This being used as a location points configuration rather than tier points config. I would rename to `LocationPointsConfiguration`